packages/ds-react-core/tsconfig.json

```diff
@@ -3,12 +3,21 @@
   "compilerOptions": {
     "baseUrl": "src",
     "skipLibCheck": true,
-    "types": ["node", "react", "react-dom"]
+    "types": [
+      "node",
+      "react",
+      "react-dom",
+      "vitest/globals",
+      "@testing-library/jest-dom"
+    ]
```

Comment:
Adding types needed for vitest (`vitest/globals`, `@testing-library/jest-dom`).
These are excluded from the build config.